py/nightwatch/run.py

```diff
@@ -455,11 +455,24 @@ def make_plots(infile, basedir, preprocdir=None, logdir=None, rawdir=None, camer
     if (preprocdir is not None):
         #- plot preprocessed images
         downsample = 4
-
-        for camera in cameras:
-            input = os.path.join(preprocdir, "preproc-{}-{:08d}.fits".format(camera, expid))
-            output = os.path.join(expdir, "preproc-{}-{:08d}-4x.html".format(camera, expid))
-            web_plotimage.write_image_html(input, output, downsample, night)
+        
+        ncpu = get_ncpu(None)
+        input = os.path.join(preprocdir, "preproc-{}-{:08d}.fits")
+        output = os.path.join(expdir, "preproc-{}-{:08d}-4x.html")
+        
+        argslist = [(input.format(cam, expid), output.format(cam, expid), downsample, night) for cam in cameras]
+    
+        if ncpu > 1:
+            pool = mp.Pool(ncpu)
+            pool.starmap(web_plotimage.write_image_html, argslist)
+            pool.close()
+            pool.join()
+            
+        else:
+            for camera in cameras:
+                input = os.path.join(preprocdir, "preproc-{}-{:08d}.fits".format(camera, expid))
+                output = os.path.join(expdir, "preproc-{}-{:08d}-4x.html".format(camera, expid))
+                web_plotimage.write_image_html(input, output, downsample, night)
```

Comment:
to make sure that the `ncpu>1` case exactly matches the non-parallelized case, let's do something like this for the "else" clause:
```
else:
    for args in arglist:
        web_plotimage.write_image_html(*args)
```
i.e. that guarantees that the arguments used by the `ncpu>1` case matches the arguments used by the `else` case without replicating the code of the `os.path.join` etc.

Otherwise fine to merge when ready.